File: poo/PHP/car.php
<?php
require_once('account.php');
class Car {
    public $id;
    public $license;
    public $driver;
    protected $passengers;

    public function __construct($license, $driver) {
        $this->license = $license;
        $this->driver = $driver;
    }
    public function printDataCar() {
        echo "Licencia: ", $this->license, "<br/>Driver: ", $this->driver->name, "<br/>Pasajeros: ", $this->passengers, "<br/>";
    }
    public function getPassengers() {
        return $this->passengers;
    }
    public function setPassengers($passengers) {
        if($passengers == 4){
            $this->passengers = $passengers;
        }
        else {
            echo "Necesita Seleccionar 4 pasajeros";
        }
    }
}
?>

File: poo/Java/Car.java
class Car {
    private Integer id;
    private String license;
    private Account driver;//Heredaré el nombre(o la clase) de account
    Integer passengers;//

    public Car(String license, Account driver) {//Parámetros Obligatorios
        this.license = license;
        this.driver = driver;
    }

    void printDataCar() {//Funcion de Imprimir los datos
        if(passengers != null){
            System.out.println("Licencia: " + license + "\nName Driver: " + driver.name + "\nPasajeros: " + passengers);
        }
    }

    public Integer getPassengers(){
        return passengers;
    }
    public void setPassengers(Integer passengers){
        if(passengers == 4){
            this.passengers = passengers;
        }
        else {
            System.out.println("Necesitas asignar 4 pasajeros");
        }
    }

    public Integer getId() {
        return id;
    }

    public void setId(Integer id) {
        this.id = id;
    }
    
}